src/components/count-badge/CountBadge.scss

```diff
@@ -36,7 +36,7 @@ $bdl-CountBadge-background: $bdl-watermelon-red;
     font-size: 11px;
     line-height: 16px;
     text-align: center;
-    background-color: $bdl-CountBadge-background;
+    background-color: #e9163c;
```

Comment:
Can we us a Box color? `$bdl-watermelon-red`?

https://opensource.box.com/box-ui-elements/storybook/?path=/story/theming-colors--all-colors

Reply:
unfortunaly not this time... we talk this with David and there is no red-color with enough contrast